example/lib/list_example.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class ListExample extends StatelessWidget {
+  const ListExample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 400,
+      height: 400,
```

Comment:
Was this leftover from testing and getting this working? I can't remember where we added a temp value for height and width to resolve the overflow error.